packages/react/src/components/ClassPrefix/index.tsx

```diff
@@ -6,10 +6,21 @@
  */
 
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { PropsWithChildren } from 'react';
 import { PrefixContext } from '../../internal/usePrefix';
 
-function ClassPrefix({ children, prefix }) {
+interface ClassPrefixProps {
+  /**
+   * @description The value used to prefix the CSS selectors
```

Comment:
The `@description` JSDoc annotation is [not supported](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#unsupported-tags)

```suggestion
   * The value used to prefix the CSS selectors
```

Additional usages of `@description` in this PR will need updated as well